frontend/src/App.tsx

```diff
@@ -30,9 +35,95 @@ const App = () => {
       .then(setApps);
   }, []);
 
+  const sortHandler = () => {
+    setSortState((sortMeth === "Due Date")? "Club Name" : "Due Date");
+  }
+
+  /**
+   * Convert date into number of format yyyymmdd
+   * @param date a valid date in the format of Month dd, yyyy
+   */
+  const dateToInt = (date: string) => {
```

Comment:
there's usually a cleaner way to do this: refer to https://stackoverflow.com/questions/13566552/easiest-way-to-convert-month-name-to-month-number-in-js-jan-01 for more info